src/ansible/roles/packages/tasks/RedHat8.yml

```diff
@@ -0,0 +1,6 @@
+- name: Enable idm module
```

Comment:
I wonder, do we really want to decouple all Red Hat releases? If so, why didn't you define a file for RHEL7 and RHEL9?

Reply:
This change was needed due to the magic done in the custom filter. 
The filter matches first distribution+number and then distribution without number so when there was RedHat and RedHat7 it always matched RedHat7.
When I put in RedHat7 it was incorrectly matching so Pavel said to do it this way.
I removed RHEL 7 from this MR to push forward but it might return in future.

Reply:
Understood. Thanks for the explanation